doozerlib/distgit.py

```diff
@@ -2083,6 +2083,7 @@ def _get_csv_file_and_refs(self, csv_config):
         for cand in ref_candidates:
             if os.path.isfile(cand):
                 refs = cand
+                break
```

Comment:
It seems to me that adding this `break` statement just interrupts the loop as soon as the first candidate is found. Without it, it would overwrite `refs` as new candidates come along. 
Could you provide some context about what we're solving here?